test/xpu_api/numerics/complex.number/complex.transcendentals/log.pass.cpp

```diff
@@ -124,7 +124,7 @@ void test_edges()
 
 ONEDPL_TEST_NUM_MAIN
 {
-    test<float>();
+    IF_DOUBLE_SUPPORT(test<float>())
```

Comment:
The problem lies in the usage of the macro `INFINITY`.
It seems that this macro ex[ands into an expression, during the evaluation of which a `double` type appears.